BRAINSConstellationDetector/src/landmarksConstellationCommon.cxx

```diff
@@ -154,9 +154,9 @@ CreatedebugPlaneImage(SImageType::Pointer referenceImage, const std::string & de
 }
 
 SImageType::Pointer
-CreatedebugPlaneImage(SImageType::Pointer               referenceImage,
-                      const RigidTransformType::Pointer MSPTransform,
-                      const std::string &               debugfilename)
+CreatedebugPlaneImage(SImageType::Pointer                 referenceImage,
```

Comment:
Should referenceImage be passed as reference?

Reply:
Yes, it is covered in PR[#501](https://github.com/BRAINSia/BRAINSTools/pull/501/files). This PR only converted constant pass-by-value parameters to constant pass-by-reference parameters. The other PR converts non-constant pass-by-value parameters to constant pass-by-reference parameters. The changes are quite similar, so it likely could have been one PR, but they are a bit different so I separated them.